packages/host/src/AppDrawer.tsx

```diff
@@ -21,9 +21,9 @@ import {
   useMediaQuery,
   useTheme,
   useTranslation,
+  NavLink,
```

Comment:
Changed from `AppNavLink` -> `NavLink` because there wasn't enough offroad diffing in this PR